src/Axios/Axios.ts

```diff
@@ -1,25 +1,5 @@
-import axios, { AxiosResponse } from 'axios';
+import axios from 'axios';
 
-import { AuthService } from '../Api/client/AuthService';
-
-export const instance = axios.create({
-    baseURL: process.env.REACT_APP_API_URL,
-    headers: {
-        'Access-Control-Allow-Origin': '*',
-        'Access-Control-Allow-Methods': 'GET,PUT,POST,DELETE'
-    }
+export const request = axios.create({
+    baseURL: process.env.REACT_APP_API_URL
 });
```

Comment:
everything looks good but on this file I think we need @Veryyapeee opinion

Reply:
CORS is set up on the server side. Doing this on client side doesn't change anything